http_service/tests/test_push_schedules.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import logging
+import time
+
+import pytest
+from rq.exceptions import NoSuchJobError
+
+from bugbug_http import app
+
+# { <job id>: <result key> }
+JOBS = {}
+
+
+class RedisMock:
```

Comment:
There is a `fakeredis` package for mocking too, but this is good too for me.

Reply:
It's simple enough and avoids another dependency :). Feel free to fix in a follow-up if you like.